core/domain/feedback_services.py

```diff
@@ -446,9 +670,20 @@ def clear_feedback_message_references(user_id, exploration_id, thread_id):
 
 
 def _get_all_recipient_ids(exploration_id, thread_id, author_id):
-    """Fetches all valid recipient ids from exploration and thread.
+    """Fetches all authors of the exploration excluding the given author and
+    all the other recipients.
 
-    Returns a tuple with batch_recipients and other_recipients.
+    Args:
+        exploration_id: str.
+        thread_id: str.
+        author_id: str. One author of the given exploration_id.
+
+    Returns:
+         tuple of (batch_recipients, other_recipients)
+            batch_recipients: set(str). Set of the authors who own this
+                exploration excluding the given author.
+            other_recipients:  set(str). Set of the recipients of this
+                exploration excluding all the authors.
```

Comment:
Actually this is incorrect. It should be: The user_ids of the other participants in this thread, excluding owners of the exploration and the given author. There's no such thing as the "recipient of an exploration".